bot/app/database/db_objects.py

```diff
@@ -0,0 +1,234 @@
+        except sqlite3.IntegrityError as e:
+            logging.error(e)
+        
+        curs.close()
+        conn.close()
+
+class PlaysAt:
+
+    """
+    Instance represents Artist-Event connection (aka playsAt Table)
+    """
+
+    def __init__(self, artist_name, event_name, date):
+        
+        self.artist_name = artist_name
+        self.event_name = event_name
+        self.date = date
+
+    def check_for_artist(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+        
+        curs.execute("select * from Artists where name=?", (self.name, ))
+        artist = curs.fetchone()
+        
+        if artist:
+            print("Corresponding artist found! Check: \n", artist)
+            answ = input("Is this the right entry? [Y/n]" )
+            if answ == "Y":
+                pass
+            else:
+                self.name = input("Please re-enter artist-name: ")
+                check_for_artist()
+        else: 
+            self.name = input("Please re-enter artist-name: ")
+            self.check_for_artist()
+
+        curs.close()
+        conn.close()
+
+    def check_for_event(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+        
+        curs.execute("select * from Events where eName=? and date=?", (self.event_name, self.date))
+        event = curs.fetchone()
+
+        if event:
+            print("Corresponding event found! Check: \n", event)
+            answ = input("Is this the right entry? [Y/n]" )
+            if answ == "Y":
+                pass
+            else:
+                self.e_name = input("Please re-enter event-name: ")
+                check_for_event()
+        else: 
+            self.e_name = input("Please re-enter event-name: ")
+            self.check_for_event()
+
+        curs.close()
+        conn.close()
+
+    def insert_plays_at(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        try:
+            curs.execute("""
+                insert into playsAt values (?,?,?)""",
+                (self.artist_name, self.event_name, self.date))
+            self.conn.commit()
+        except sqlite3.IntegrityError as e:
+            logging.error(e)
+        
+        curs.close()
+        conn.close()
+
+class User:
+
+    """Instance of the User relation:
+
+    Arguments:
+        * u_id(str): User-ID, same as the User-ID which Telegram uses
+        * name(str): First name of the user
+        * is_bot(bool): True if the account is a bot
+    """
+
+    def __init__(self, u_id, name, is_bot):
+
+        self.u_id = u_id
+        self.name = name
+        self.is_bot = is_bot
+
+    def user_exists(u_id):
+
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        curs.execute("select * from Users where uID=?", (self.u_id, ))
+        user = curs.fetchone()
+
+        curs.close()
+        conn.close()
+
+        return user if not None 
```

Comment:
This should be  `return user is not None`, so replace `if` with `is`.

Right now you would return the user object, or `None` if it doesn't exist. But you want to return `True` or `False` right?

Reply:
fixed